lte/gateway/deploy/roles/magma/tasks/main.yml

```diff
@@ -517,3 +517,12 @@
     force: yes
     remote_src: yes
   when: full_provision
+
+# TODO: preburn this and add it under MME(OAI) dependencies
+- name: Install SQLite3 C++ library
+  retries: 5
+  apt:
+    state: latest
+    pkg:
+      - libsqlite3-dev
```

Comment:
@nstng , @vroon2703  note that "Install common Magma dependencies" is a preburn action, so moving this library to that action will only be effective if we update the `magma-dev` box base image. It will not install the library if we just destroy and re-build the VM locally or in CI.

Reply:
Since updating the magma-dev box image is out of scope for @vroon2703 's project plan, my suggestion would be to keep the install step for sqlite3 package as a `full_provision` action for this PR and someone from DevOps team can take over the box update task separately.

Reply:
@ssanadhya nice catch, I overlooked the "when" - sorry @vroon2703 you need your change from before with the comment that this dependency should be added to "Install common Magma dependencies" when the next preburn imaged is created.

Reply:
Done!